README.md

```diff
@@ -23,17 +23,16 @@ A single interface to use different guardrail models.
 
 </div>
 
-## [Documentation](https://mozilla-ai.github.io/any-guardrail/)
 
-## Motivation
+`any-guardrail` provides a unified interface for AI safety guardrails, for example, letting you detect toxic content, PII leakage, jailbreak attempts, and other risks in LLM inputs and outputs. Switch between different guardrail providers, both encoder-based (discriminative) and decoder-based (generative) models like Llama Guard and ShieldGemma, without changing your code.
```

Comment:
Nit: let's remove PII leakage for now. We don't have any models guardrailing against PII leakage atm.